scalafmt-tests/src/test/resources/default/Try.stat

```diff
@@ -64,3 +64,9 @@ try { foo()  } catch handler
 try {
   foo()
 } catch handler
+<<< #350 long single line
+val template: String = try source.mkString finally source.close()
+>>>
+val template: String =
+  try source.mkString
+  finally source.close()
```

Comment:
Please try to test

```
<<< ONLY #350 long single line with catch and finally
val template: String = try source.mkString catch { case e if NonFatal(e) => log.error("failed to make string", e) } finally source.close()
>>>
val template: String =
  try source.mkString
  catch { case e if NonFatal(e) => log.error("failed to make string", e) }
  finally source.close()
```

On your branch it formats as
```
val template: String =
  try source.mkString
  catch { case e if NonFatal(e) => log.error("failed to make string", e) } finally source
    .close()
```

Reply:
thank you, will try to fix